### PR TITLE
Support stop framework api in k8s rest server

### PR DESCRIPTION
**Pull request by abuccts**
Support stop framework api in k8s rest server.

### CODE REVIEW

File: src/rest-server/src/routes/v2/job.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ router.route('/:frameworkName')
   /** GET /api/v2/jobs/:frameworkName - Get job */
   .get(controller.get);
 
+router.route('/:jobName/executionType')
+  /** PUT /api/v2/jobs/:frameworkName/executionType - Start or stop job */
+  .put(token.check, controller.execute);
```
Comment (yqwang-ms) [inline src/rest-server/src/routes/v2/job.js]: patch instead of put

File: src/rest-server/src/models/v2/job/k8s.js [hunk 2]
```diff
@@ -423,6 +423,34 @@ const put = async (frameworkName, config, rawConfig) => {
...
+  try {
+    response = await axios({
+      method: 'patch',
+      url: launcherConfig.frameworkPath(encodeName(frameworkName)),
+      headers: {
+        'Content-Type': 'application/merge-patch+json',
+      },
+      data: {
+        spec: {
+          executionType: `${executionType.charAt(0)}${executionType.slice(1).toLowerCase()}`,
```
Comment (yqwang-ms) [inline src/rest-server/src/models/v2/job/k8s.js]: Case:

image

File: src/rest-server/src/routes/v2/job.js (hunk 1 above)
Comment (abuccts) [reply src/rest-server/src/routes/v2/job.js]: this is rest server's api, used `PATCH` in k8s apiserver's request

File: src/rest-server/src/models/v2/job/k8s.js (hunk 2 above)
Comment (abuccts) [reply src/rest-server/src/models/v2/job/k8s.js]: right, this is CamelCase

**Pull request merged**